planemo/galaxy/activity.py

```diff
@@ -722,9 +714,53 @@ def get_dict_from_workflow(gi, workflow_id):
     return gi.workflows.export_workflow_dict(workflow_id)
 
 
-def _wait_for_invocation(ctx, gi, history_id, workflow_id, invocation_id, polling_backoff=0):
+def wait_for_invocation_and_jobs(
```

Comment:
Should we also have this implemented in BioBlend?
Once it's in a BioBlend release, we could simplify the code here (and maybe in some Galaxy tests as well).

Reply:
I don't know, there's many ways to do it, and none seem optimal ... I'd probably rather include subworkflow invocation jobs on the Galaxy side ?